docs/get_started/plan_deployment/traditional_deployment/cluster_consideration/clus_maint.md

```diff
@@ -1,30 +1,32 @@
 # Cluster maintenance
 
-Maintaining HCL Portal in a cluster typically means applying corrective services \(fix packs and interim fixes\) or updating the software release level on each node in the cluster.
+Maintaining HCL Portal in a cluster typically means applying fix packs and interim fixes or updating the software release level on each node in the cluster.
 
-Fixes are classified as "minor" if they do not update the underlying HCL Portal databases or require version upgrades to other supporting software such as databases servers or IBM® WebSphere® Application Server. Most of the HCL Digital Experience service packs are not considered minor and might require the use of a separate installation procedure to ensure 24x7 availability.
+Fixes are considered "minor" if they don’t update the underlying HCL Portal databases or require version upgrades to supporting software such as databases servers or IBM® WebSphere® Application Server.
 
-Instructions for applying corrective service to a HCL Portal cluster are provided with the corrective service package. Before you apply any maintenance, analyze any user impact. Ensure that you are able to provide uninterrupted service \(also referred to as 24x7 availability\), even during the maintenance phase.
+Most HCL Digital Experience service packs are not considered minor and might require the use of a separate installation procedure to ensure 24x7 availability.
+
+Instructions for applying maintenance service to an HCL Portal cluster are provided with the service package. Before you apply maintenance, analyze user impact. Ensure that you can provide uninterrupted service \(also referred to as 24x7 availability\), even during the maintenance phase.
```

Comment:
Instructions to apply maintenance service to an HCL DX cluster are provided with the service package. Before you apply maintenance, analyze user impact. Ensure that you can provide uninterrupted service (also referred to as 24x7 availability), even during the maintenance phase.